x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/exploration_query_bar/exploration_query_bar.tsx

```diff
@@ -59,6 +60,31 @@ export const ExplorationQueryBar: FC<ExplorationQueryBarProps> = ({
 
   const searchChangeHandler = (q: Query) => setSearchInput(q);
 
+  const regex = useMemo(() => new RegExp(`${filters?.columnId}\s?:\s?(true|false)`, 'g'), []);
```

Comment:
should it have `filters?.columnId` is the dependencies array?

Reply:
Yes, it should - good catch 🙏

Reply:
Added in a349738dc5dddfb51907e5d04aa22126df55ff9e